tests/phpunit/tests/post/wpCountAttachments.php

```diff
@@ -28,6 +28,7 @@ public function test_wp_count_attachments_should_cache_the_result() {
 		$expected = wp_count_attachments( $mime_type );
 		$actual   = wp_cache_get( $cache_key, 'counts' );
 
-		$this->assertEquals( $expected, $actual );
+		$this->assertSame( $expected->$mime_type, $actual->$mime_type );
+		$this->assertSame( $expected->trash, $actual->trash );
```

Comment:
This change is doing more than changing from `assertEquals()` to `assertSame()`. I think the change to test the parameters should go into a separate ticket. The assertion change though can go in https://core.trac.wordpress.org/ticket/56800.

@Rahmon how would you like to handle this? Maybe open a new ticket and reference Trac 56800?

Reply:
Yes, I would like to handle this.

In this case, I didn't just change from `$this->assertEquals( $expected, $actual );` to `$this->assertSame( $expected, $actual );` because the test failed since we're comparing two objects. 

So, I'll create a new ticket and reference Trac 56800.

Thanks for your review @hellofromtonya  :raised_hands: